xir/parser.py

```diff
@@ -275,69 +225,98 @@ def application_stmt(self, args):
         }
         return Statement(name, params, wires, **stmt_options)
 
-    def obs_stmt(self, args):
-        """Observable statement. Defined inside an observable definition.
-
-        Returns:
-            ObservableStmt: object containing statement data
+    @v_args(inline=True)
+    def obs_def(self, name, params, wires, statements):
         """
-        pref = simplify_math(args[0])
-        terms = args[1]
-        return ObservableStmt(pref, terms, use_floats=self.use_floats)
+        Creates an observable definition from text of the form:
```

Comment:
(minor) the first sentence should be a single line (ending with a `.`) on the same line as the `"""`. This can then be followed by a more detailed description. I realize that I didn't follow this convention fully before either.
```python
"""Creates an observable definition and adds it to the program.

Creates an observable definition from text of the form:
...
"""
```
No a big deal here, since these docstring currently aren't rendered in the readthedocs.

Reply:
gotcha, didn't know this. thanks for the info!